AK/Bitmap.h

```diff
@@ -275,24 +277,25 @@ class Bitmap {
         size_t max_region_size = 0;
 
         while (true) {
-            // going to find the next block which is bigger than current
+            // going to find the next block which is bigger than curr
             auto length_of_found_range = find_next_range_of_unset_bits(start, max_region_size + 1, max_length);
             if (length_of_found_range.has_value()) {
-                // if we have value, it's definitely begger than current
+                // if we have value, it's bigger than current
                 max_region_start = start;
                 max_region_size = length_of_found_range.value();
                 start += max_region_size;
             } else {
-                // there are no ranges whach are bigger than current, so let's finish
+                // there are no ranges which are bigger than current,
+                // so let's finish
                 break;
             }
         }
 
         found_range_size = max_region_size;
-        if (!max_region_size) {
-            return {};
+        if (max_region_size) {
+            return max_region_start;
         }
-        return max_region_start;
+        return {};
```

Comment:
There are all kinds of unrelated changes in this commit. (Unrelated to the commit message.)
Please squash them into the relevant previous commit instead so that commits are small and precise.